samples/node/ConsoleApp/README.md

```diff
@@ -0,0 +1,21 @@
+# Node Console App
+This sample demonstrates a simple console app to be run in node. 
+
+Restore dependencies
+--------------------
+```
+npm install
+dotnet restore
+```
+Build the app
+-------------------
+This will start the Fable server and invoke webpack to build and bundle the app.
+```
+dotnet fable npm-run build
```

Comment:
I am no expert on webpack nor rollup. The reason I chose webpack here is becuase it seems to be the default way we do stuff in Fable so the choice is arbitrary. However, it would be nice if could send a PR with a similar sample using Rollup